core/controlplane/config/templates/cluster.yaml

```diff
@@ -1311,8 +1311,10 @@ experimental:
   # Enable audit log for apiserver. Recommended when `rbac` is enabled.
   auditLog:
     enabled: false
-    maxage: 30
-    logpath: /dev/stdout
+    logPath: /var/log/kube-apiserver-audit.log
```

Comment:
Ref https://github.com/kubernetes-incubator/kube-aws/pull/579 when this was changed in the code. Never seen stdout working well so far with regard to https://github.com/kubernetes-incubator/kube-aws/issues/419.